graphql-database-manager-dynamo/src/main/java/com/fleetpin/graphql/database/manager/dynamo/DynamoDb.java

```diff
@@ -28,15 +26,15 @@
 
 import com.fleetpin.graphql.database.manager.util.CompletableFutureUtil;
 import com.fleetpin.graphql.database.manager.util.TableCoreUtil;
-import org.reactivestreams.Subscriber;
-import org.reactivestreams.Subscription;
-import software.amazon.awssdk.core.async.SdkPublisher;
+import com.google.common.base.Throwables;
+
 import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient;
 import software.amazon.awssdk.services.dynamodb.model.*;
-import software.amazon.awssdk.utils.CompletableFutureUtils;
 
 public final class DynamoDb extends DatabaseDriver {
-    private final AttributeValue GLOBAL = AttributeValue.builder().s("global").build();
+    private static final AttributeValue REVISION_INCREMENT = AttributeValue.builder().n("1").build();
+
+	private final AttributeValue GLOBAL = AttributeValue.builder().s("global").build();
```

Comment:
Just a nitpick, but adjust the indentation.

Reply:
fixed. Had tabs not spaces :(